src/sentry/integrations/opsgenie/client.py

```diff
@@ -1,9 +1,19 @@
 from __future__ import annotations
 
+from typing import Any, Mapping, Union
+
+from requests import Response
+
 from sentry.integrations.client import ApiClient
+from sentry.models import Integration
+from sentry.services.hybrid_cloud.integration.model import RpcIntegration
+from sentry.shared_integrations.response.base import BaseApiResponse
 
 OPSGENIE_API_VERSION = "v2"
 
+# for typing purposes
+BaseApiResponseX = Union[BaseApiResponse, Mapping[str, Any], Response]
+
```

Comment:
Rather than declare this, it's often better to just import the type from the file with the original signature so that we don't repeat ourselves.